templates/zerver/features.html

```diff
@@ -301,7 +301,7 @@
                     <div class="block">
                         <h4>{% trans %}Desktop apps{% endtrans %}</h4>
                         <p>{% trans %}Prefer Zulip in its own window and rich, OS-level notifications?
-                          Enjoy <a href="/apps">Zulip on your desktop</a>.{% endtrans %}</p>
+                          Enjoy <a href="https://zulipchat.com/apps">Zulip on your desktop</a>.{% endtrans %}</p>
```

Comment:
Nope, but REALMS_HAVE_SUBDOMAINS is the name of the feature in question, and I thought might be helpful for explaining how these variables work.